grassroots-backend/src/grassroots-shared/util/FetchSuccessOrThrow.ts

```diff
@@ -0,0 +1,15 @@
+interface FetchResponse<T, E> {
+  data?: T;
+  error?: E;
+  response: Response;
+}
+
+export function fetchSuccessOrThrow<T, E>(fetchResult: FetchResponse<T, E>): T {
```

Comment:
This could be pulled into a separate patch, and possible by made a nicer wrapper around grassrootsAPI. See https://github.com/gpo/grassroots/issues/155

Reply:
Done.